autorest/codegen/models/lro_operation.py

```diff
@@ -89,37 +89,62 @@ def has_optional_return_type(self) -> bool:
         return False
 
     def get_poller_path(self, async_mode: bool) -> str:
-        extension_name = "poller-async" if async_mode else "poller-sync"
+        extension_name = "poller-{}sync".format("a" if async_mode else "")
```

Comment:
Does it really improve the readability? :p

Reply:
made some fixes for readability